ports/cpu-features/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cpu-features",
-  "version-string": "0.4.1",
+  "version-string": "0.6.0",
```

```suggestion
  "version": "0.6.0",
```
It would be better to use `version `here. Please refer to this doc. https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md